saleor/product/models/base.py

```diff
@@ -197,7 +198,7 @@ class Meta:
         app_label = 'product'
 
     def __str__(self):
-        return self.name or self.sku
+        return self.display_product()
```

Comment:
Why are these two separate methods? Do we ever need to pass `attributes` to `display_*` instead of doing a `prefetch_related`?